src/index.hot.js

```diff
@@ -1,19 +1,20 @@
+/* eslint global-require: 0 */
+import 'babel-polyfill';
```

Comment:
for production mode, I will separate main.js and vendor.js

For develop mode, I think I should keep which are consistent?

(but it seems nobody use that on develop mode @@?)